### PR TITLE
i18n(es): updates `overrides`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- Closes # N/A 
- What does this PR change? Give us a brief description.
Updates `overrides` based on these commits:
  - ca0678ca556d739bda9648edc1b79c764fdea851
  - 61493e55f1a80362af13f98d665018376e987439

### CODE REVIEW

File: docs/src/content/docs/es/reference/overrides.md [hunk 1]
```diff
@@ -51,6 +51,12 @@ Etiqueta de idioma BCP-47 para la configuración regional de esta página, por e
 
 La ruta base en la que se sirve un idioma. `undefined` para los slugs de idioma raíz.
 
+#### `siteTitle`
+
+**Type:** `string`
```
Comment (HiDeoo) [inline docs/src/content/docs/es/reference/overrides.md:56]:
```suggestion
** Tipo:** `string`
```

To match the other props

File: docs/src/content/docs/es/reference/overrides.md [hunk 2]
```diff
@@ -331,6 +337,12 @@ Componente que contiene el elemento `<h1>` de la página actual.
 
 Las implementaciones deben asegurarse de establecer `id="_top"` en el elemento `<h1>` como en la implementación predeterminada.
 
+#### `DraftContentNotice`
+
+**Default component:** [`DraftContentNotice.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/DraftContentNotice.astro)
```
Comment (HiDeoo) [inline docs/src/content/docs/es/reference/overrides.md:342]:
```suggestion
**Componente por defecto:** [`DraftContentNotice.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/DraftContentNotice.astro)
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the update :thumbsup:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). Left 2 small suggestions, but otherwise, this looks good to me.

**Comment by dreyfus92** [comment]
Nice catch, thank you! 😁

**Review by HiDeoo — APPROVED** [review]
Thanks for the fast fixes.

**Pull request merged**